code/game/machinery/alarm.dm

```diff
@@ -302,11 +302,18 @@
 
 	if(old_danger_level!=danger_level)
 		apply_danger_level()
+		if(mode == AALARM_MODE_SCRUBBING && danger_level == ATMOS_ALARM_DANGER)
+			if(pressure_dangerlevel == ATMOS_ALARM_DANGER)
+				mode = AALARM_MODE_OFF
+				if(temperature_dangerlevel == ATMOS_ALARM_DANGER && cur_tlv.max2 <= environment.temperature)

```

Comment:
I'ed put this outside of if(mode == AALARM_MODE_SCRUBBING), as a atmos tech I like to set toxins to contaminated at round start, to save time 😄